### ISSUE TITLE
"Error while loading the archive" -- Pre-trained Weights

### ISSUE THREAD

**Issue by thanif** [ChaoningZhang/DeepPTZ#1]
Hi,

Thank You very much for this repo. I downloaded the weights for evaluation but was unable to extract the file, getting the following error: 
![1](https://user-images.githubusercontent.com/32266960/91418701-f0f5b280-e86b-11ea-8b90-b06237dc04b3.png)

I think, there might be a problem while uploading the file and re-uploading it might solve the issue. 

Kindly provide feedback. 

Thanks

**Comment by ChaoningZhang** [comment]
Thank you very much for your interest in our work.
After you download the model, please put the model in a folder named pretrained_model and then run the command "bash test.sh". Also Please the put the downloaded dataset also in the right directory.
You do not need to extract the file for evaluation. Hope this answered your question.
If you need extra help, do not hesitate to ask

**Comment by thanif** [comment]
Thank You very much for the response.
Another question was about the test set as there are only two sets available as Train and Val, Is Val the test set or not?

**Comment by ChaoningZhang** [comment]
Val set is the same as Test set. Similar to ImageNet that only has Train set and Val Set, it is not quite meaningful to distinguish Test dataset and Val dataset in this context. I indeed generated another independent dataset as test dataset, which delivers the same performance. Hope this answers your question.

**Comment by thanif** [comment]
Thanks.
